### DISCUSSION TITLE
Cache Limits

### DISCUSSION THREAD

**Discussion by APixelVisuals** [discordjs/discord.js#6137] [Q&A] [answered]
It looks like the docs have a [cache with limits](https://discord.js.org/#/docs/main/master/class/Options?scrollTo=s-cacheWithLimits) method. Are there any plans to allow developers to choose what gets cached? For example, caching guild emojis could be irrelevant for your bot, and keeping that data would be wasting memory.

**Comment by advaith1** [comment, marked as answer]
you already can, just set a manager to 0:
- #6013
